packages/remark-lint-no-shell-dollars/index.js

```diff
@@ -59,6 +57,7 @@ var reason = 'Do not use dollar signs before shell commands'
 // See: <https://github.com/github/linguist/blob/40992ba/lib/linguist/languages.yml#L4984>
 var flags = [
   'sh',
+  'shell',
```

Comment:
this seems to be an unrelated change?

Reply:
Yes, I can undo this, if you want. Looks weird for me to make a new branch, a new commit, a new PR just for one word.

Reply:
@ChristianMurphy, so?

Reply:
Please split this into it's own PR, thanks!

Reply:
Hmm. I don’t think that’s a good idea. The comment above the list contains an URL to what GitHub sees as shell code, which matches the current list. I don’t think we should deviate from that.

Reply:
> Hmm. I don’t think that’s a good idea. The comment above the list contains an URL to what GitHub sees as shell code, which matches the current list. I don’t think we should deviate from that.

Thank you, I didn't notice that. But it's strange: GitHub catches `shell` well, you can check it yourself. I don't know why, and I don't want to dive into `linguist` sources, but I guess it takes root key (`Shell:` for our case) as option too. **Or** GitHub actually uses an another version of `linguist`.

> Please split this into its own PR, thanks!

OK, removed, sorry.